grafana/grafana
# Loki: backend: use json-field

the grafana-backend now provides a native JSONField that can be used in dataframes. this pull request changes the code to use it. note that now we don't need to do any extra work in javascript, so some code could be removed.

i also had to fix a couple error-message-strings and unit-tests strings, those are in a separate commit.

how to test:
1. make sure you have the `lokiBackendMode` feature flag enabled
2. run a normal loki query. make sure you receive labels. the easiest way to check is to click on a line, and when the details open up, make sure you see something reasonable in the "Log labels" section.
